### PR TITLE
Add custom pulley card view 

### PR DESCRIPTION
**Pull request by lucyxubroad**
### Overview 
Made the media view controller with a card view that can be pulled up like a Pulley.

### Related Issues
#19 

### Screenshots/GIFs
custom-pulley

### Note
Still have to ask Cindy about height preferences for views and update buttons once Cindy gets them to me.

### CODE REVIEW

File: Flick/Controllers/MediaCardViewController.swift [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+//  Flick
+//
+//  Created by Lucy Xu on 7/7/20.
+//  Copyright © 2020 flick. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+enum CardState { case expanded, collapsed}
```
Comment (haiyingweng) [inline Flick/Controllers/MediaCardViewController.swift:12]: nit: missing space after collapsed

File: Flick/Controllers/MediaViewController.swift [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+
+    // MARK: - Private View Vars
+    private var mediaCardViewController: MediaCardViewController!
+    private let mediaImageView = UIImageView()
+    private let saveMediaButton = UIButton()
+    private var visualEffectView: UIVisualEffectView!
+
+    // MARK: - Private Data Vars
+    private var animationProgressWhenInterrupted: CGFloat = 0
+    private let buttonSize = CGSize(width: 72, height: 72)
```
Comment (haiyingweng) [inline Flick/Controllers/MediaViewController.swift:22]: we used 44 for other buttons

File: Flick/SceneDelegate.swift [hunk 3]
```diff
@@ -41,7 +41,7 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
             // User is logged out.
             rootViewController = loginViewController
         }
-        let navigationController = UINavigationController(rootViewController: rootViewController)
+        let navigationController = UINavigationController(rootViewController: MediaViewController())
```
Comment (haiyingweng) [inline Flick/SceneDelegate.swift:44]: move it back to rootViewController?

File: Flick/Controllers/MediaViewController.swift (hunk 2 above)
Comment (lucyxubroad) [reply Flick/Controllers/MediaViewController.swift:22]: I checked the designs just now and it says 36? Maybe we can ask Cindy

**Pull request merged**